artview/core/common.py

```diff
@@ -50,7 +50,7 @@ def ShowQuestion(msg):
     return response
 
 
-def ShowLongText(msg):
+def ShowLongText(msg, modal=True):
```

Comment:
modal dialog (executed with `exec_()`) lock in the position and just continue the code when closed, other windows may also lock/ get faded etc.
Modales dialog (executed with `show()`) continue execution normally and the window can stay open without affecting the others.